### PR TITLE
datatrails: include description of related metrics tab

### PR DESCRIPTION
**Pull request by darrenjaneczek**
**What is this feature?**

image

Added description to "related metrics" tab.
It is possible to add descriptions to all tabs now.

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by torkelo** [comment]
Similarity in name??

**Comment by catherineymgui** [comment]
I like Torkel's suggestion ^ we can update and it's good

File: public/app/features/trails/MetricScene.tsx [hunk 1]
```diff
@@ -113,7 +113,12 @@ export class MetricScene extends SceneObjectBase<MetricSceneState> {
 const actionViewsDefinitions: ActionViewDefinition[] = [
   { displayName: 'Overview', value: 'overview', getScene: buildMetricOverviewScene },
   { displayName: 'Breakdown', value: 'breakdown', getScene: buildBreakdownActionScene },
-  { displayName: 'Related metrics', value: 'related', getScene: buildRelatedMetricsScene },
+  {
+    displayName: 'Related metrics',
+    value: 'related',
+    getScene: buildRelatedMetricsScene,
+    description: 'Related metrics are correlated by similarity of their names',
```
Comment (darrenjaneczek) [inline public/app/features/trails/MetricScene.tsx:120]:
```suggestion
    description: 'Related metrics are correlated by similarity in name',
```

**Comment by darrenjaneczek** [comment]
> I like Torkel's suggestion ^ we can update and it's good

Ok, changed here:
- https://github.com/grafana/grafana/pull/84525/commits/121023a3b718e8c17bb92936a78827052161bff7

**Review by catherineymgui — APPROVED** [review]

**Comment by torkelo** [comment]
@catherineymgui @darrenjaneczek it was a question not a suggestion, I do not understand this description, it is not correct. 

We show metric names that match current label filters, not metric names that has any name similarity with the current name

**Comment by catherineymgui** [comment]
There was additional work done by @adrapereira to sort the related metrics by similarity to the name (cannot find the PR). 
We could update this 

"Related metrics are correlated by similarity in name and labels"?

**Comment by torkelo** [comment]
@catherineymgui that is just the sorting. 

Maybe we should rename the tab to "Other metrics" with a description of "Switch to other relevant metrics for current label filters"

**Comment by torkelo** [comment]
but let's keep the word "Related metrics" but a description like "Relevant metrics for current label filters"

**Comment by catherineymgui** [comment]
Okay, that is fine to me. `Relevant metrics based on current label filters`

**Comment by darrenjaneczek** [comment]
Ok @catherineymgui 

Changed description to:
>Relevant metrics based on current label filters

Kept "Related metrics" as per:
- https://github.com/grafana/grafana/pull/84525#issuecomment-2000678972

Confirm that we no longer want to rename to "Other metrics" as previously suggested:
- https://github.com/grafana/grafana/pull/84525#issuecomment-2000076334

**Review by torkelo — APPROVED** [review]

**Pull request merged**